src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/20210302153400_AddSystemToMcPkgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Migrations
+{
+    public partial class AddSystemToMcPkgs : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "System",
+                table: "McPkgs",
+                type: "nvarchar(max)",
```

Comment:
Is there no limits in any nvarchar columns in IPO?

Reply:
Yes there are, but have not added it here. Will add a max length of 40 to systems (do not see any systems longer than this in QP at least), and for mcPkg and commPkg nos.